components/widget/ChatInput.tsx

```
@@ -116,18 +122,86 @@ export default function ChatInput({
+      setSelectedFiles([]);
     } else if (SCREENSHOT_KEYWORDS.some((keyword) => input.toLowerCase().includes(keyword))) {
       setShowScreenshot(true);
     }
   }, [input]);
 
   useEffect(() => {
     if (screenshot?.response) {
-      handleSubmit(screenshot.response);
+      // Get any pending attachments that were selected before screenshot
+      const pendingAttachments = (window as any).pendingAttachments || [];
+      delete (window as any).pendingAttachments;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)